src/ralph/admin/templatetags/dashboard_tags.py

```diff
@@ -54,6 +56,18 @@ def get_user_equipment_to_accept_tile_data(user):
     }
 
 
+def get_user_simcard_to_accept_tile_data(user):
```

Comment:
Could you provide some test for this function? Hint: mock ``get_simcards_to_accept``to simplify testing.

Reply:
Test passed, and pushed to git.